src/app/views/change-password/changePasswordRequest.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
```

Comment:
Note, haven't reviewed this file other than it compiles as this is to be constructed in a future task

Reply:
not sure about this comment. Ta